src/Tizen.NUI/src/public/ViewProperty/BackgroundExtraData.cs

```diff
@@ -50,17 +47,6 @@ internal Rectangle BackgroundImageBorder
             set => backgroundImageBorder = new Rectangle(value);
         }
 
-        /// <summary></summary>
-        internal Vector4 CornerRadius { get; set; }
-
-        /// <summary></summary>
-        internal Vector4 CornerSquareness { get; set; }
-
-        /// <summary>
-        /// Whether the CornerRadius value is relative (percentage [0.0f to 0.5f] of the view size) or absolute (in world units).
-        /// </summary>
-        internal VisualTransformPolicyType CornerRadiusPolicy { get; set; } = VisualTransformPolicyType.Absolute;
-
```

Comment:
Bad news.

Their are some codes who using those values internally. actually, FLUX.
Until those code changed, we need to keep those properties.

```
[Obsolete("Do not use this, that is deprecated in API13."]
internal Vector4 CornerRadius { get; set; }

[Obsolete("Do not use this, that is deprecated in API13."]
internal Vector4 CornerSquareness { get; set; }

[Obsolete("Do not use this, that is deprecated in API13."]
internal Vector4 CornerRadiusPolicy { get; set; } = VisualTransformPolicyType.Absolute;
```

Reply:
Do we have to set and get actual properties?
Or can I just return zero and leave only the form.